cross/build-rootfs.sh

```diff
@@ -16,7 +16,7 @@ __InitialDir=$PWD
 __BuildArch=arm
 __UbuntuArch=armhf
 __UbuntuRepo="http://ports.ubuntu.com/"
-__LLDB_Package="lldb-3.6-dev"
+__LLDB_Package=""
```

Comment:
I believe this change broke our Arm32 Ubuntu build:

```
2017-02-16T00:36:40.4286270Z -- Looking for ucol_setMaxVariable - not found
2017-02-16T00:36:40.4374030Z CMake Error at src/ToolBox/SOS/lldbplugin/CMakeLists.txt:60 (message):
2017-02-16T00:36:40.4388130Z   Cannot find lldb-3.5, lldb-3.6, lldb-3.8 or lldb-3.9.  Try installing
2017-02-16T00:36:40.4406950Z   lldb-3.6-dev (or the appropriate package for your platform)
2017-02-16T00:36:40.4414420Z 
2017-02-16T00:36:40.4420130Z 
2017-02-16T00:36:40.4432910Z -- Configuring incomplete, errors occurred!
2017-02-16T00:36:40.4445300Z See also "/root/coreclr/bin/obj/Linux.arm.Checked/CMakeFiles/CMakeOutput.log".
2017-02-16T00:36:40.4456490Z See also "/root/coreclr/bin/obj/Linux.arm.Checked/CMakeFiles/CMakeError.log".
2017-02-16T00:36:40.4468580Z ~/coreclr
2017-02-16T00:36:40.4479530Z Failed to generate CoreCLR component build project!
```

If we want to nullify lddb package for arm64, it should be done conditionally and not absolutely.